src/client/views/settings-view/measure-modal/measure-modal.tsx

```diff
@@ -25,36 +25,43 @@ import { SvgIcon } from '../../../components/svg-icon/svg-icon';
 import { FormLabel } from '../../../components/form-label/form-label';
 import { Button } from '../../../components/button/button';
 import { ImmutableInput } from '../../../components/immutable-input/immutable-input';
+import { ImmutableDropdown } from '../../../components/immutable-dropdown/immutable-dropdown';
 import { Modal } from '../../../components/modal/modal';
 
 import { Measure } from '../../../../common/models/index';
 
+import { MEASURE_EDIT as LABELS } from '../utils/labels';
+
 
 export interface MeasureModalProps extends React.Props<any> {
   measure?: Measure;
   onSave?: (measure: Measure) => void;
   onClose?: () => void;
+  isCreating?: boolean;
```

Comment:
rather than a `boolean` this might work better as `mode: "creating" | "editing"`